core/src/test/scala/org/apache/spark/sql/delta/UpdateScalaSuite.scala

```diff
@@ -60,6 +59,23 @@ class UpdateScalaSuite extends UpdateSuiteBase  with DeltaSQLCommandTest {
       Row(100, 101) :: Row(100, 101) :: Row(3, 30) :: Row(4, 40) :: Nil)
   }
 
+  test("update usage test - with space in path") {
+    val tmpDirForTest = System.getProperty("java.io.tmpdir").concat("/test with space/")
+    val dirFile = Utils.createTempDir(tmpDirForTest, "spark")
+    val dir = dirFile.toString
+    val df = Seq((1, 10), (2, 20), (3, 30), (4, 40)).toDF("key", "value")
+    val writer = df.write.format("delta").mode("append")
+    writer.save(dir)
+    spark.sql("CREATE TABLE test USING DELTA LOCATION '" + dir + "'")
+    spark.sql("UPDATE test SET key = key + 10")
+    spark.sql("UPDATE test SET key = key + 10 WHERE true")
```

Comment:
We don't need to write multiple queries for update as it doesn't increase the test coverage. Instead, could you try to add a `merge` command and a `delete` command? `getTouchedFile` is used by update, delete and merge. Technically, the update command should cover it. But it would be great to test all of them, so that if we create a different `getTouchedFile` method for one of these commands in future, this test will prevent us from breaking it.